.github/actions/debug-info/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Install RcppDeepState with its dependendcies
+Rscript -e "install.packages(c('Rcpp', 'RcppArmadillo', 'RInside', 'data.table', 'qs', 'nc', 'xml2'), repos='http://cran.us.r-project.org')"
+Rscript -e "install.packages('./', repos = NULL, type='source', verbose=TRUE)"
```

Comment:
explicitly writing the dependencies here is repetitive with the info from DESCRIPTION, right?
You should instead use devtools::install_dev_deps() which examines DESCRIPTION and installs whatever is listed there.

Reply:
That's right! I replaced this docker based workflow with a new one based on [r-lib/actions](https://github.com/r-lib/actions). In this way all the dependencies are correctly installed based on the `DESCRIPTION` file. This new workflow has been implemented in the commit 05e0b06